--- libtorrent-rasterbar-1.1.4/include/libtorrent/export.hpp	2017-03-09 06:51:52.000000000 +0300
+++ /tmp/SBo/libtorrent-rasterbar-1.1.4/include/libtorrent/export.hpp	2017-08-23 02:04:45.178968279 +0300
@@ -33,19 +33,19 @@
 #ifndef TORRENT_EXPORT_HPP_INCLUDED
 #define TORRENT_EXPORT_HPP_INCLUDED
 
-#if !defined(BOOST_COMPILER_CONFIG) && !defined(BOOST_NO_COMPILER_CONFIG)
-#  include <boost/config/select_compiler_config.hpp>
-#endif
-#ifdef BOOST_COMPILER_CONFIG
-#  include BOOST_COMPILER_CONFIG
-#endif
+//#if !defined(BOOST_COMPILER_CONFIG) && !defined(BOOST_NO_COMPILER_CONFIG)
+//#  include <boost/config/select_compiler_config.hpp>
+//#endif
+//#ifdef BOOST_COMPILER_CONFIG
+//#  include BOOST_COMPILER_CONFIG
+//#endif
 
-#if !defined(BOOST_PLATFORM_CONFIG) && !defined(BOOST_NO_PLATFORM_CONFIG)
-#  include <boost/config/select_platform_config.hpp>
-#endif
-#ifdef BOOST_PLATFORM_CONFIG
-#  include BOOST_PLATFORM_CONFIG
-#endif
+//#if !defined(BOOST_PLATFORM_CONFIG) && !defined(BOOST_NO_PLATFORM_CONFIG)
+//#  include <boost/config/select_platform_config.hpp>
+//#endif
+//#ifdef BOOST_PLATFORM_CONFIG
+//#  include BOOST_PLATFORM_CONFIG
+//#endif
 
 // backwards compatibility with older versions of boost
 #if !defined BOOST_SYMBOL_EXPORT && !defined BOOST_SYMBOL_IMPORT
